### PR TITLE
Port configuration and Slicer get stuck when connection fails

### PR DESCRIPTION
**Pull request by che85**
fixes issue #1 
fixes issue #2

### CODE REVIEW

**Comment by jcfr** [comment]
Looks good.

Cc: @lassoan

**Comment by jcfr** [comment]
Thanks for the contribution :smile:

File: PyDevRemoteDebug/PyDevRemoteDebug.py [hunk 1]
```diff
@@ -448,8 +473,8 @@ def connect(self):
 
     # Connect to the debugger
     try:
-      pydevd.settrace('localhost', port=self.portNumber, stdoutToServer=True, stderrToServer=True, suspend=False)
-    except Exception, e:
+      pydevd.settrace('localhost', port=port, stdoutToServer=True, stderrToServer=True, suspend=False)
```
Comment (lassoan) [inline PyDevRemoteDebug/PyDevRemoteDebug.py]: port=port may be valid but looks a bit strange. I think this would be more clear:
port = portNumber

**Comment by lassoan** [comment]
Thanks. Looks good to me, too.

I've never experienced Slicer crashing related to connecting to the debugger, it just waits indefinitely until a debugger becomes available. If a debugger is not attached successfully then on the next start of Slicer the auto-connect to debugger is automatically disabled. Also, auto-connect can only be enabled if you've connected to the debugger successfully. So, overall I think the worst scenario is that if somebody sets up a debugger, enables auto-connect, and then somehow forgets how to start the debugger again - but even in this case Slicer hangs only a single time.

**Pull request merged**